pkg/engine/internal/imageverifier.go

```diff
@@ -12,7 +13,9 @@ import (
 	"github.com/kyverno/kyverno/pkg/config"
 	"github.com/kyverno/kyverno/pkg/cosign"
 	engineapi "github.com/kyverno/kyverno/pkg/engine/api"
+	ct "github.com/kyverno/kyverno/pkg/engine/context"
```

Comment:
Why is this duplicated? Please remove and use the existing declaration.

Reply:
Yeah, sure